config/blazegraph/blazegraph.production.properties

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
These are necessary for BG to work correctly for the indexing file structure and logging.. we need them locally for development, but these are reference/basic examples. Implementation on the server will be similar, but should be configured seperately, ran with 'god', and more tuned properties files.